det3d/datasets/argoverse/argoverse_dataset.py

```diff
@@ -0,0 +1,106 @@
+from det3d.datasets.argoverse.argoverse_common import *
+from det3d.datasets.custom import PointCloudDataset
+from det3d.datasets.registry import DATASETS
+
+
+@DATASETS.register_module
+class ArgoverseDataset(PointCloudDataset):
+
+    def __init__(self, root_path, pipeline=None, test_mode=False):
```

Comment:
Thanks for this PR, Chen. Can we add types here on the input arguments?

Reply:
Sure! Updating them in the upcoming commit.